### PR TITLE
fix(unit_test): bump current enterprise version to 2022.2

### PR DESCRIPTION
**Pull request by KnifeyMoloko**
Unit test will fail because we have a newer version of enterprise than 2022.1. Bumping the version in `test_enterprise()` to 2022.2

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x]  All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by soyacz — APPROVED** [review]

**Comment by fgelcer** [comment]
@scylladb/qa-maintainers , who is handling the backports?

**Comment by fruch** [comment]
Aren't we still waiting for @roydahan approval ? 

as for 2022.2 and 5.1, I think we'll be FFing them for a while, and not backporting.

**Comment by fgelcer** [comment]
> Aren't we still waiting for @roydahan approval ?
> 
> as for 2022.2 and 5.1, I think we'll be FFing them for a while, and not backporting.

this is fixing a unit test... and yes, for 2022.2 and 5.1 we should be FF'ing

**Comment by fruch** [comment]
> > Aren't we still waiting for @roydahan approval ?
> > as for 2022.2 and 5.1, I think we'll be FFing them for a while, and not backporting.
> 
> this is fixing a unit test... and yes, for 2022.2 and 5.1 we should be FF'ing

and in general, I think whom merges should take care of the backports as needed.

**Pull request merged**
